### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by stevezau** [jfinkels/flask-restless#642]
Hi,

Is there a new release planned? there are a few bug fixes I need which were fixed last Nov but no release since.

Thanks

**Comment by Natureshadow** [comment]
Yeah, please just push 1.0.0b2 to PyPI. It would make our alpha version of Veripeditus installable. We do need 1.0.0b2, 1.0.0b1 doe not work for us, and pip are removing the dependency links feature.
